### DISCUSSION TITLE
Intel NCS2 ASIC Compatibility

### DISCUSSION THREAD

**Discussion by MaxwellDPS** [openai/whisper#227] [General]
Howdy All!

Amazing work! So im trying to test a [NCS2](https://www.intel.com/content/www/us/en/developer/tools/neural-compute-stick/overview.html) with Ubuntu 22.04 and python3.9. I have OpenVino inplace and now on running with the device MYRAD I get the error below. Is there a way to fix this by loading through the python interpreter.

Thanks in advance!

![image](https://user-images.githubusercontent.com/11069376/193469618-aaa37379-dfa4-4e01-8a98-8828c6f46b6e.png)

**Comment by jodfie** [comment]
bump also have this issue.

**Comment by richardburleigh** [comment]
You can try changing [this line](https://github.com/openai/whisper/blob/9f70a352f9f8630ab3aa0d06af5cb9532bd8c21d/whisper/__init__.py#L106) to something like:

```
checkpoint = torch.load(fp, map_location="CPU")
checkpoint.to("MYRIAD")
```
I don't have a device to test, but this solved issues mapping storage on an Apple M1 GPU.

**Comment by etlweather** [comment]
Did you get this to work by chance?
